### PR TITLE
api rest: add option lookup_recursive for getMyEntitiesMaster

### PR DESCRIPTION
**Pull request by garnier-quentin**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | not

With the webservice, we cannot get the complete list of entities. So an option, it's added to get the complete list (recursive entities).

### CODE REVIEW

**Comment by trasher** [comment]
Documentation (apirest.md file) may be updated according to the change

**Comment by garnier-quentin** [comment]
You re right. It's done.

File: inc/api.class.php [hunk 1]
```diff
@@ -305,15 +305,29 @@ protected function changeActiveEntities($params = []) {
     *
     * @return array of entities (with id and name)
     */
-   protected function getMyEntities() {
+   protected function getMyEntities($params = []) {
```
Comment (orthagh) [inline inc/api.class.php]: corresponding docblock missing

**Comment by garnier-quentin** [comment]
Change done! Thanks :)

**Pull request merged**